set_env.sh

```diff
@@ -0,0 +1,7 @@
+export SNORKELHOME="$HOME/Documents/snorkel"
```

Comment:
Hardcoded path not ideal. Why are setting these environment variables necessary?